sw/device/tests/sim_dv/pwrmgr_normal_sleep_all_wake_ups.c

```diff
@@ -53,14 +53,18 @@ bool test_main(void) {
     LOG_INFO("POR reset");
 
     for (size_t i = 0; i < PWRMGR_PARAM_NUM_WKUPS; ++i) {
+      if (kDeviceType != kDeviceSimDV &&
+          i == PWRMGR_PARAM_ADC_CTRL_AON_WKUP_REQ_IDX) {
+        continue;
+      }
       LOG_INFO("Test %d begin", i);
       execute_test(i, /*deep_sleep=*/false);
       check_wakeup_reason(i);
       LOG_INFO("Woke up by source %d", i);
       cleanup(i);
       LOG_INFO("clean up done source %d", i);
     }
-
+    LOG_INFO("Done");
```

Comment:
when the test returns true, the ottf will already print "pass" / "fail" to the console, which can be picked up manually, or using the UartConsole struct and a match expression, like here: https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/tests/chip/power_virus/src/main.rs;drc=f6ce5b96271e414b01c103e568d392aafbcc1f7a;l=29

Perhaps we should turn lines 29--61 [here](https://cs.opensource.google/opentitan/opentitan/+/master:sw/host/tests/chip/power_virus/src/main.rs;drc=f6ce5b96271e414b01c103e568d392aafbcc1f7a;l=29) into an opentitanlib testutil to make this test exit scheme reusable across tests?

Reply:
I don't think that will work: the power virus test is only waiting for a pass or fail, plus timeouts and Ctrl-C. This test is waiting for that and all sorts of other messages that will trigger some activity. Am I missing something?

Reply:
However, I can see how to straighten up the test body to better handle the regex and ExitStatus... In some other PR...